requirements.txt

```diff
@@ -4,4 +4,4 @@ google-cloud-bigquery==1.11.1
 confuse==0.5.0
 slackclient==1.3.1
 requests==2.22.0
-
+functions-framework==3.*
```

Comment:
Please, make sure to add also on Pipfile, and update the Pipfile.lock.

Reply:
pipfile and pipfile.lock was update
[1d81727](https://github.com/kieras/zombie-projects-watcher/pull/63/commits/1d81727f4a688ed955c8590d8f61fa4bc631bab4)